.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
+}
```

Comment:
I think all of the .vs files shouldn't be checked in, as they are configuration settings from your editor. You probably want to add
```
.vs/*
```
to our gitignore so we don't add it in further commits. For now you want to remove these files from the commit and update the pr...